tests/sentry/integrations/slack/webhooks/commands/test_link_team.py

```diff
@@ -114,6 +122,17 @@ def test_link_team_sufficient_role_through_team(self):
         data = self.send_slack_message("link team", user_id=OTHER_SLACK_ID)
         assert "Link your Sentry team to this Slack channel!" in get_response_text(data)
 
+    @responses.activate
+    def test_link_team_as_team_admin(self):
+        """
+        Test that when a user who is a team admin attempts to link a team we allow it.
+        """
+        self.login_as(self.team_admin_user)
+        link_user(self.team_admin_user, self.idp, slack_id=OTHER_SLACK_ID)
+
+        data = self.send_slack_message("link team", user_id=OTHER_SLACK_ID)
+        assert "Link your Sentry team to this Slack channel!" in get_response_text(data)
+
 
 @region_silo_test(stable=True)
 class SlackCommandsUnlinkTeamTest(SlackCommandsLinkTeamTestBase):
```

Comment:
Is there a test for normal user failing to link/unlink? If not please add one.

Reply:
Yes there's a test called `test_link_team_insufficient_role` that fails for regular org members